examples/faker-import.ts

```diff
@@ -0,0 +1,4 @@
+import Faker from 'faker'
```

Comment:
i'd add two example files. one with bad one (you can disable by eslint-disable) and one with good

Reply:
after ignoring the types/ folder and making the lint tests pass, the new eslint rule still does not fail here somehow. any ideas why @rostislav-simonik-plc ?